test/e2e/resize_nodes.go

```diff
@@ -38,9 +38,12 @@ import (
 
 const serveHostnameImage = "gcr.io/google_containers/serve_hostname:1.1"
 
+const timeoutWaitNodeReady = 2 * time.Minute
+const timeoutWaitNodeNotReady = 2 * time.Minute
```

Comment:
Thanks; renaming to resizeNodeNotReadyTimeout (adding the prefix and following the naming pattern you suggested).